context-and-compliance/03-environments.md

```diff
@@ -0,0 +1,33 @@
+# 3 - Environments
+
+## 3.1 Application has corresponding environments and hosting
+
+### 3.1.1 `testing` environment
+
+#### Context & justification
+
+The `testing` environment, as the name suggests, is an environment for testing. This is where all development/testing is done. Hence, it should only be accessible to engineers and for internal delivery teams. Self-explanatory.
```

Comment:
Might be worth specifying "manual testing for the development team".

I've seen cases where development environments end up using resources from a hosted test environment. It's not pretty.

Reply:
Resolved in https://github.com/madetech/productionisation/pull/58/commits/8fd653e94c4c8d997663f95bdf49801fea5f08c8.